### DISCUSSION TITLE
Limit on number of automatic exports vs one huge export

### DISCUSSION THREAD

**Discussion by erazlogo** [retorquere/zotero-better-bibtex#2904] [Q&A] [answered]
Is there a limit on the number of collections with automatic export "on change"?
I had two "project" collections 7000+ items and 10000+ items, and almost everything in my Zotero belongs to one or the other. This made Zotero very slow. So I am trying to break down into many subject collections and export those--but there can be potentially 50+ collections with automatic export scheduled. Will that slow down Zotero also? Thank you!

**Comment by retorquere** [comment, marked as answer]
This slowing down will scale roughly linearly with the number of items being exported; separate collections will help if they contain few overlapping items. The work I'm doing on #2315 will address these problems alongside addressing memory use. You can try the build I dropped there. Disable the cache, and you should see improvement (after the first export, because the cache needs to fill initially).

**Reply by erazlogo** [reply]
Thank you, the build seems to have helped! I have the latest Zotero 7 by the way.
